app/controllers/application_controller.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def aplication
+    puts 'defualt typo'
```

Comment:
__[RuboCop]__

Rails/Output: Do not write to stdout. Use Rails's logger if you want to log.


<!-- Comment by [Sider](https://sider.review) -->